library/debug.c

```diff
@@ -246,7 +246,7 @@ void mbedtls_debug_print_mpi( const mbedtls_ssl_context *ssl, int level,
     else
     {
         int n;
-        for( n = ( bitlen - 1 ) / 8; n >= 0; n-- )
+        for( n = (int) ( bitlen - 1 ) / 8; n >= 0; n-- )
```

Comment:
Nitpick: why do the conversion at that particular point?
```c
for( n = (int) ( ( bitlen - 1 ) / 8 ); n >= 0; n-- )
```
would be more intuitive or
```c
for( n = ( (int) bitlen - 1 ) / 8; n >= 0; n-- )
```
would give a better hint to the compiler (in case `int` is smaller than `size_t`).